docs/advanced_usage.rst

```diff
@@ -289,4 +290,84 @@ Command Line Options
 
 .. cc-command-line-options::
 
+.. _choice-variables:
+   
+Choice Variables (1.1+)
+-----------------------
+
+Choice variables provide different choices when creating a project. Depending on an user's choice
+the template renders things differently.
+
+Basic Usage
+~~~~~~~~~~~
+
+Choice variables are regular key / value pairs, but with the value being a list of strings.
+
+For example, if you provide the follwing choice variable in your ``cookiecutter.json``::
+
+   {
+       "license": ["MIT", "BSD-3", "GNU GPL v3.0", "Apache Software License 2.0"]
+   }
+
+you'd get the following choices when running Cookiecutter::
+
+   Select license:
+   1 - MIT
+   2 - BSD-3
+   3 - GNU GPL v3.0
+   4 - Apache Software License 2.0
+   Choose from 1, 2, 3, 4 [1]:		
+
+Depending on an user's choice, a different license is rendered by Cookiecutter. 
+
+The above ``license`` choice variable creates ``cookiecutter.license``, which
+can be used like this::
+
+  {%- if cookiecutter.license == "MIT" -%}
+  # Possible license content here
+  
+  {%- elif cookiecutter.license == "BSD-3" -%}
+  # More possible license content here
+
+Cookiecutter is using `Jinja2's if conditional expression <http://jinja.pocoo.org/docs/dev/templates/#if>`_ to determine the correct license.
+
+The created choice variable is still a regular Cookiecutter variable and can be used as::
 
+  License
+  -------
+
+  Distributed under the terms of the `{{cookiecutter.license}}`_ license,
+  
+Overwriting Default Choice Values
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Choice Variables are overwritable using a :ref:`user-config` file.
```

Comment:
changed this a little bit different. I agree that the sentence is redudant, but I think it is important to remind a possible reader that this setting is done in the configuration file.